bip-0328/_base58.py

```diff
@@ -107,7 +107,7 @@ def get_xpub_fingerprint(s: str) -> bytes:
     """
     Get the parent fingerprint from an extended public key
 
-    :param s: The extended pubkey
+    :param xpub: The extended pubkey
```

Comment:
It looks like this line should either remain as-is, or be updated to the name "xpub" in the code of this function.

Reply:
In case I was unclear, the above comment was for this line only, not for https://github.com/bitcoin/bips/pull/1808#discussion_r2023074403.